.github/workflows/tests.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
       matrix:
         # os: ["windows-latest", "ubuntu-latest", "macos-latest"]
         os: ["ubuntu-latest"]
-        python-version: ["3.7", "3.8", "3.9"]
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
I think we can do this too 

```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Reply:
Not yet. We need numba/numba#8304

Reply:
Ah, I see. In that case it seems strange to me that we're adding a `ci/environment-3.11.yaml` file.

Reply:
Good point. I can remove it from this PR.